### ISSUE TITLE
Slowness Effect messes with the FOV 

### ISSUE THREAD

**Issue by Muqsit** [pmmp/PocketMine-MP#1551]
### Issue description

- Expected result: FOV reverts back to normal after slowness effect is removed/expires.
- Actual result: FOV does not revert back to normal until you rejoin. _This cannot be reproduced in vanilla_.

### Steps to reproduce the issue

1. /give <yourself> slowness 1 100
2. Your FOV wont revert back.

### OS and versions

* PocketMine-MP: 48a7627
* PHP: 7.2RC6
* Server OS: Ubuntu 14.04 LTS 64-bit
* Game version: Win10

### Plugins
- Test on a clean server without plugins: Issue still exists

**Comment by IrishPacks** [comment]
I remember having this issue on earlier versions(dating back to 0.15). Running fixed it for me.

**Comment by dktapps** [comment]
The only way I can reproduce this is by using excessive amplifiers so that the player's speed decreases to zero. Is that what you're describing?

**Comment by Muqsit** [comment]
Yes. In vanilla, the speed does get put to zero when you use amplifier >= 7 but the FOV reverts back to normal once slowness goes away.

**Comment by dktapps** [comment]
The problem arises here: https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/entity/Effect.php#L474 - due to the dividend of the operation being zero. It's capped (is that the right word?) at zero when adding the effect and can't go below that.

In vanilla I think this is handled using attribute modifiers which effectively change the output value of an attribute, instead of modifying its base value directly. This is not implemented in PocketMine-MP.

Not sure how best to fix this - I don't think there's any real non-hacky way to do it.

**Comment by djdisodo** [comment]
https://github.com/pmmp/PocketMine-MP/blob/2d3ce9e8b027161c9d8b4145c94a3813d1ab9497/src/pocketmine/entity/Effect.php#L361
this sets negative value if effectlevel is >=7

**Comment by djdisodo** [comment]
https://github.com/djdisodo/PocketMine-MP/tree/patch-5
